localization/imu_augmentor/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ find_package(catkin2 REQUIRED COMPONENTS
 
 catkin_package(
   INCLUDE_DIRS include ${GTSAM_INCLUDE_DIR} ${GLOG_INCLUDE_DIRS} 
-  LIBRARIES ${PROJECT_NAME} ${GLOG_LIBRARIES} ${GTSAM_LIBRARIES} 
+  LIBRARIES ${PROJECT_NAME} ${GLOG_LIBRARIES} ${GTSAM_LIBRARIES} gtsam 
```

Comment:
This should be part of gtsam libraries?

Reply:
For some reason it's not, not sure what the best way to account for that is but for some packages if I don't include it there I get linker errors

Reply:
You could make a FindGSTAM in the cmake folder which fixes this. Will leave it to your judgement whether it's worth the effort.